apps/src/containedLevels.js

```diff
@@ -74,7 +74,7 @@ export function runAfterPostContainedLevel(fn) {
   if (postState === PostState.None) {
     throw new Error('Shouldnt call runAfterPostContainedLevel before postContainedLevelAttempt');
   }
-  if (PostState.Finished) {
+  if (postState === PostState.Finished) {
```

Comment:
this seems like a pretty significant bug; has this been affecting all contained levels?